projects/kserve/testing/test_e2e.py

```diff
@@ -151,12 +151,22 @@ def deploy_and_test_models_e2e():
     logging.info("Wait 60s for Prometheus to restart collecting data ...")
     time.sleep(60)
 
+    using_longevity = config.ci_artifacts.get_config("tests.e2e.longevity.enabled")
+
     try:
         deploy_models_concurrently()
 
-        test_models_concurrently()
+        if not using_longevity:
+            test_models_concurrently()
+            test_models_sequentially(locally=False)
+        else:
+            repeats = config.ci_artifacts.get_config("tests.e2e.longevity.repeats")
+            delay = config.ci_artifacts.get_config("tests.e2e.longevity.delay")
+            for i in range(repeats):
+                test_models_concurrently()
```

Comment:
here, you'll need that, for matrix-benchmarking visualization:
```suggestion
                with env.NextArtifactDir(f"longevity_{i}"):
                    with open(env.ARTIFACT_DIR / "settings.longevity.yaml", "w") as f:
                        yaml.dump(dict(index=i), f, indent=4)
                    test_models_concurrently()
```